nx/lib/nx.ex

```diff
@@ -2481,9 +2481,22 @@ defmodule Nx do
         [Inf, NaN]
       >
 
+  If the input is a numerical constant instead of a tensor, this is an alias to `Nx.tensor(number, type: type)`.
+  In the example below, notice how precision is only lost if we pass a type which can't represent the numerical input.
```

Comment:
```suggestion
  If the input is a numerical constant instead of a tensor, this is an
  alias to `Nx.tensor(number, type: type)`. In the example below,
  notice how precision is only lost if we pass a type which can't
  represent the numerical input:
```